docs/widgets.md

```diff
@@ -1910,6 +1911,40 @@ Display info from the Gluetun VPN container public IP API. This can show the IP
 
 ---
 
+### Drone.io Builds
+
+Display the last builds from a (drone.io)(https://www.drone.io] instance. 
+
+<p align="center"><img width="380" src="https://i.ibb.co/nQM3BXj/Bildschirm-foto-2023-01-07-um-01-31-45.png" /></p>
+
+#### Options
+
+**Field** | **Type** | **Required** | **Description**
+--- | --- | --- | ---
+**`host`** | `string` |  Required | The histname of the drone.io instance
+**`apiKey`** | `string` |  Required | The API key (https://<drone-instance>/account)
+**`limit`** | `integer` |  Optional | Limit the amounts of listed builds.
+
+#### Example
+
+```yaml
+- type: DroneIo
```

Comment:
Very minor one, but the other widgets ID is in lowercase, with dashes replacing spaces. So something like `drone-io` / `droneio` would be more consistent.